sunpy/timeseries/sources/goes.py

```diff
@@ -64,63 +64,87 @@ class XRSTimeSeries(GenericTimeSeries):
-
-        ax.plot_date(dates, self.to_dataframe()['xrsa'], '-',
-                     label=r'0.5--4.0 $\AA$', color='blue', lw=2, **kwargs)
-        ax.plot_date(dates, self.to_dataframe()['xrsb'], '-',
-                     label=r'1.0--8.0 $\AA$', color='red', lw=2, **kwargs)
-
-        ax.set_yscale("log")
-        ax.set_ylim(1e-9, 1e-2)
-        ax.set_title(title)
-        ax.set_ylabel('Watts m$^{-2}$')
-        ax.set_xlabel(datetime.datetime.isoformat(self.to_dataframe().index[0])[0:10])
-
-        ax2 = ax.twinx()
+        axes.plot_date(
+            dates, self.to_dataframe()["xrsa"], "-", label=r"0.5--4.0 $\AA$", color="blue", lw=2, **kwargs
+        )
+        axes.plot_date(
+            dates, self.to_dataframe()["xrsb"], "-", label=r"1.0--8.0 $\AA$", color="red", lw=2, **kwargs
+        )
+
+        axes.set_yscale("log")
+        axes.set_ylim(1e-9, 1e-2)
+        axes.set_title(title)
+        axes.set_ylabel("Watts m$^{-2}$")
+        axes.set_xlabel(datetime.datetime.isoformat(self.to_dataframe().index[0])[0:10])
+
+        ax2 = axes.twinx()
         ax2.set_yscale("log")
         ax2.set_ylim(1e-9, 1e-2)
-        labels = ['A', 'B', 'C', 'M', 'X']
+        labels = ["A", "B", "C", "M", "X"]
         centers = np.logspace(-7.5, -3.5, len(labels))
         ax2.yaxis.set_minor_locator(mticker.FixedLocator(centers))
         ax2.set_yticklabels(labels, minor=True)
         ax2.set_yticklabels([])
 
-        ax.yaxis.grid(True, 'major')
-        ax.xaxis.grid(False, 'major')
-        ax.legend()
+        axes.yaxis.grid(True, "major")
+        axes.xaxis.grid(False, "major")
+        axes.legend()
 
         # TODO: display better tick labels for date range (e.g. 06/01 - 06/05)
-        formatter = matplotlib.dates.DateFormatter('%H:%M')
-        ax.xaxis.set_major_formatter(formatter)
+        formatter = matplotlib.dates.DateFormatter("%H:%M")
+        axes.xaxis.set_major_formatter(formatter)
+
+        axes.fmt_xdata = matplotlib.dates.DateFormatter("%H:%M")
 
-        ax.fmt_xdata = matplotlib.dates.DateFormatter('%H:%M')
+        return axes
+
+    @peek_show
+    def peek(self, title="GOES Xray Flux", **kwargs):
+        """
+        Displays the GOES XRS light curve by calling `~sunpy.timeseries.sources.goes.XRSTimeSeries.plot`.
+
+        .. plot::
+
+            import sunpy.timeseries
+            import sunpy.data.sample
+            ts_goes = sunpy.timeseries.TimeSeries(sunpy.data.sample.GOES_XRS_TIMESERIES, source='XRS')
+            ts_goes.peek()
+
+        Parameters
+        ----------
+        title : `str`, optional
+            The title of the plot. Defaults to "GOES Xray Flux".
+        **kwargs : `dict`
+            Additional plot keyword arguments that are handed to `~matplotlib.axes.Axes.plot`
+            functions.
+        """
+        fig, ax = plt.subplots()
+        self.plot(axes=ax, title=title, **kwargs)
         fig.autofmt_xdate()
 
```

Comment:
```suggestion

```

Reply:
Do we not need an empty line between the method and it's return statement?

Reply:
No?

Reply:
Oh wow okay, I had been doing that for every PR that I had made until now. I'll make this change.

Reply:
It's just a style choice.